README.md

```diff
@@ -76,10 +77,47 @@ use the following command:
 `pip install -i https://testpypi.python.org/pypi alertlogic-cli`
 
 
-In order to manipulate previous releases(delete, hide, etc.) PyPi web interface to be used:
+In order to manipulate previous releases (delete, hide, etc.) you need to use PyPi web interface:
 https://pypi.python.org/pypi - for the production
 https://testpypi.python.org/pypi - for the testpypi
 
-
-General documentation is listed here:
+More info:
 https://packaging.python.org/distributing/
+
+
+## How to add new commands:
+
+1. Add a subdirectory in alertlogiccli/commands/
+
+2. Write at least 1 subcommand, they must be classes that inherit from Command
```

Comment:
maybe it's worth adding an `examples` directory and put a working skeleton of the command module.